src/core/socket_info.c

```diff
@@ -1184,6 +1189,16 @@ static int build_iface_list(void)
 		nll = 0;
 		p = buf;
 		while(1) {
+			// This (and the loop above in get_flags) is
+			// broken in more than one aspect. For example,
+			// if a receive is shorter than nlmsghdr,
+			// nlp->nlmsg_type will point at garbage.
+			// Proper fix for this (and static buffer sizing
+			// problem) would be to use libnl.
```

Comment:
This comment does not belong in the source code.

If there is still any issue left after this commit, then report it in the issue tracker. The developers do not scan the code for comments pointing at possible issues.

Reply:
Removed.